### PR TITLE
FIX #62

### LINKED ISSUE #62: -K flag ignored
Recent builds ignore the -K flag and ask you to enter path to the keyfile.

Command line flags should override.

On entering through the prompt, the input does not properly expand ~ and concats to CWD.

### PR DESCRIPTION
**Pull request by aricart**
FIX issue where -K was reset in favor of expected keypath
FIX ~ paths entered in interactive where the ~ are not expanded

### CODE REVIEW

File: cmd/common.go [hunk 1]
```diff
@@ -128,10 +128,15 @@ func ReadJson(fp string, v interface{}) error {
 }
 
 func Read(fp string) ([]byte, error) {
-	afp, err := filepath.Abs(fp)
+	hfp, err := homedir.Expand(fp)
```
Comment (derekcollison) [inline cmd/common.go]: This needed if we do Abs below?

File: cmd/signerparams.go [hunk 2]
```diff
@@ -42,23 +42,23 @@ func (p *SignerParams) Edit(ctx ActionCtx) error {
...
-	ks := sctx.KeyStore
-	switch p.kind {
-	case nkeys.PrefixByteOperator:
-		KeyPathFlag = ks.GetOperatorKeyPath(sctx.Operator.Name)
-	case nkeys.PrefixByteAccount:
-		KeyPathFlag = ks.GetAccountKeyPath(sctx.Account.Name)
-	case nkeys.PrefixByteCluster:
-		KeyPathFlag = ks.GetClusterKeyPath(sctx.Cluster.Name)
+
+	if KeyPathFlag == "" {
```
Comment (derekcollison) [inline cmd/signerparams.go]: If this overrides others maybe do this first and return so its clearer what we are doing?

File: cmd/store/keys.go [hunk 3]
```diff
@@ -301,11 +302,15 @@ func Match(pubkey string, kp nkeys.KeyPair) bool {
 }
 
 func keyFromFile(path string) (nkeys.KeyPair, error) {
-	path, err := filepath.Abs(path)
+	hfp, err := homedir.Expand(path)
```
Comment (derekcollison) [inline cmd/store/keys.go]: I thought Abs would expand relative but maybe not? What is homedir? Is this another dependency?

Comment (aricart) [reply cmd/store/keys.go]: We already had the dependency - yes - abs doesn't resolve ~.

File: cmd/signerparams.go (hunk 2 above)
Comment (aricart) [reply cmd/signerparams.go]: It cannot some other operation could have set the default - (setting the account name via a flag, etc). So this check is delayed until the last possible moment during interactive.

Comment (derekcollison) [reply cmd/signerparams.go]: But flags override everything correct?

**Pull request merged**